src/common/marlin_server.cpp

```diff
@@ -480,16 +480,24 @@ void marlin_server_print_resume(void) {
 }
 
 void marlin_server_print_reheat_start(void) {
-    if ((marlin_server.print_state == mpsPaused) && (marlin_server_print_reheat_ready() == 0)) {
+    if ((marlin_server.print_state == mpsPaused) && marlin_server_print_reheat_ready()) {
         thermalManager.setTargetHotend(marlin_server.resume_nozzle_temp, 0);
     }
```

Comment:
I think I understand why it _doesn't_ set the bed target temperature. But a comment explaining that would still be useful.